notebooks/00_quick_start/sar_aml_compute_movielens.ipynb

```diff
@@ -0,0 +1,477 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<i>Copyright (c) Microsoft Corporation. All rights reserved.</i>\n",
+    "\n",
+    "<i>Licensed under the MIT License.</i>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Train SAR Single Node on MovieLens with Azure Machine Learning (Python, CPU)\n",
+    "\n",
+    "This notebook provides an exmaple of how to train SAR on remote compute resources. Details and discussions of SAR can be found in [SAR Python CPU Movielens](https://github.com/Microsoft/Recommenders/blob/master/notebooks/00_quick_start/sar_movielens.ipynb) notebook. \n",
```

Comment:
We should provide more details on why AzureML would be useful here (i.e. easier to use larger amounts of data due to scaling)

Reply:
Added to top of renamed notebook sar_movielens_with_aml.ipynb